server/routes/login.js

```diff
@@ -0,0 +1,41 @@
+const {Chef,Customer} = require('../models/index')
+const express = require('express');
+const router = express.Router();
+const _ = require('lodash');
+const bcrypt = require('bcrypt');
+const Joi = require('joi');
+const jwt = require('jsonwebtoken');
+const config = require('config');
+
+//handle request for /login
+//check user email password and return user, jwt token and user type
+router.post('/', async (req,res)=>{
+    const { error } = validate(req.body);
+    if (error) return res.status(400).send(error.details[0].message);
+    
+    let user = await Chef.findOne({ email:req.body.email });
```

Comment:
We can use the User model here instead of attempting both the Chef and Customer subtypes. 
```suggestion
    let user = await User.findOne({email: req.body.email});
```

Reply:
Changed and tested, codde is pushed in new commit